src/pages/filter/dtree/components/filter-control-dtree/filter-control-dtree.utils.ts

```diff
@@ -0,0 +1,17 @@
+import filterDtreesStore from '@store/filter-dtrees'
```

Comment:
Why create functions that do nothing but calling another functions? Call store methods directly instead.